cloudflare/resource_cloudflare_load_balancer.go

```diff
@@ -136,6 +144,149 @@ func resourceCloudflareLoadBalancer() *schema.Resource {
 	}
 }
 
+var rulesElem = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"name": {
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringLenBetween(1, 200),
+		},
+
+		"priority": {
+			Type:     schema.TypeInt,
+			Optional: true,
+			Computed: true,
+			Default:  nil,
+		},
+
+		"disabled": {
+			Type:     schema.TypeBool,
+			Optional: true,
+			Default:  false,
+		},
+
+		"condition": {
+			Type:     schema.TypeString,
+			Optional: true,
+			Default:  "",
+		},
+
+		"terminates": {
+			Type:     schema.TypeBool,
+			Optional: true,
+			Default:  false,
+		},
+
+		"overrides_tf": {
+			Type:     schema.TypeList,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+
+					"session_affinity": {
+						Type:         schema.TypeString,
+						Optional:     true,
+						Default:      nil,
+						ValidateFunc: validation.StringInSlice([]string{"", "none", "cookie", "ip_cookie"}, false),
+					},
+
+					"session_affinity_ttl": {
+						Type:         schema.TypeInt,
+						Optional:     true,
+						Default:      nil,
+						ValidateFunc: validation.IntBetween(1800, 604800),
+					},
+
+					"session_affinity_attributes": {
+						Type:     schema.TypeMap,
+						Optional: true,
+						Default:  nil,
+						Elem: &schema.Schema{
+							Type: schema.TypeString,
+						},
+					},
+
+					"ttl": {
+						Type:     schema.TypeInt,
+						Optional: true,
+						Default:  nil,
+					},
+
+					"steering_policy": {
+						Type:         schema.TypeString,
+						Optional:     true,
+						Default:      nil,
+						ValidateFunc: validation.StringInSlice([]string{"off", "geo", "dynamic_latency", "random", ""}, false),
+						Computed:     true,
+					},
+
+					"fallback_pool": {
+						Type:         schema.TypeString,
+						Optional:     true,
+						Default:      nil,
+						ValidateFunc: validation.StringLenBetween(1, 32),
```

Comment:
i'm not sure i'd bother with validating this is an ID but if you do, you would want it to be `validation.StringLenBetween(32, 32)` otherwise "abcd1234" will be valid anyway.